src/components/entity/list.vue

```diff
@@ -279,28 +317,27 @@ export default {
 
       this.$refs.table.afterUpdate(this.resolveIndex);
     },
-    showDelete(index) {
-      const entity = this.odataEntities.value[index];
+    showDelete(entity) {
       if (this.confirmDelete) {
-        this.uuidToDelete = entity.__id;
-        this.del.show({ label: entity.label });
+        this.deleteModal.show({ entity });
```

Comment:
Passing whole entity object to make it consistent with restore and Submissions deleted/restore